input/pagecontent/reference.md

```diff
@@ -1467,53 +1467,33 @@ The contents of a value set are defined by a set of rules. There are four basic
 | A single code | <code>* <span class="optional">include</span> {Coding}</code> | `* $SCT#961000205106 "Wearing street clothes, no shoes"` |
 | All codes from another value set | <code>* <span class="optional">include</span> codes from valueset {ValueSet}</code> | `* include codes from valueset http://hl7.org/fhir/ValueSet/data-absent-reason` |
 | All codes from a code system | <code>* <span class="optional">include</span> codes from system {CodeSystem}</code> | `* include codes from system http://snomed.info/sct` |
+| Codes that lie in the _intersection_ of value set(s) and (optionally) a code system | <code>* <span class="optional">include</span> codes from <span class="optional">system {CodeSystem}</span> and valueset {ValueSet1}<span class="optional">and {ValueSet2}...</span></code> | `* include codes from valueset http://hl7.org/fhir/ValueSet/units-of-time and http://hl7.org/fhir/ValueSet/age-units` |
```

Comment:
The new example stretches the table beyond the screen boundary, since the backticks create `code` elements which have the style `white-space: nowrap` applied to them.

![image](https://user-images.githubusercontent.com/22081959/167931659-b7eb24fb-5273-4072-a845-94bac69460e3.png)

Overriding the style with `white-space: normal` causes it to wrap automatically, although it might be preferable to control where the line break occurs.
```suggestion
| Codes that lie in the _intersection_ of value set(s) and (optionally) a code system | <code>* <span class="optional">include</span> codes from <span class="optional">system {CodeSystem}</span> and valueset {ValueSet1}<span class="optional">and {ValueSet2}...</span></code> | <code style="white-space: normal">* include codes from valueset http://hl7.org/fhir/ValueSet/units-of-time and http://hl7.org/fhir/ValueSet/age-units</code> |
```